appserver/admingui/web/src/main/resources/configuration/httpListenerNew.jsf

```diff
@@ -38,6 +38,8 @@
     only if the new code is made subject to such option by the copyright
     holder.
 
+    Portions Copyright [2017] [Payara Foundation] 
```

Comment:
Should be:
`Portions Copyright [2017] [Payara Foundation and/or its affiliates]`